<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>831</width>
      <height>731</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>829</width>
       <height>729</height>
      </rect>
     </property>
     <widget class="QGraphicsView" name="image_gw">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>731</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text Contents:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_content_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Templates:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="template_comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>斗方</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>扇面</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>对联</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Word space:</string>
    </property>
   </widget>
   <widget class="QSlider" name="word_space_slider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Column Space：</string>
    </property>
   </widget>
   <widget class="QSlider" name="column_space_slider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Calligraphy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="font_comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>楷体</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>宋体</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>启功体</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="set_dataset_path_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>set Dataset path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number per Column：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num_per_colu_lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
